src/main/resources/db/migration/V1_8__Create_daily_list_table.sql

```diff
@@ -10,6 +10,8 @@ CREATE TABLE IF NOT EXISTS moj_daily_list (
 , c_end_date                     TIMESTAMP WITH TIME ZONE
 , c_daily_list_id_s              CHARACTER VARYING
 , c_daily_list_source            CHARACTER VARYING
+, daily_list_content             CHARACTER VARYING
```

Comment:
Should we be going back and changing old flyway scripts - does that not confuse the flyway schema history?

Reply:
At the moment flyway is set to clean and re-create everything it is run, we left it like that because of the data model changing so frequently and we have no data to be concerned about. So this is fine right now.